src/tour/tour.js

```diff
@@ -194,6 +194,14 @@ angular.module('angular-tour.tour', [])
           scope.ttOffset = parseInt(val, 10) || tourConfig.offset;
```

Comment:
Wouldn't doing so break legacy functionality? scope.ttOffset was originally used to alter horizontal positioning on left and right aligned tourtips, but it alters vertical positioning on bottom, top, and center-top positioned tourtips in the calculatePosition() function.